static/js/components/ValidationAlert.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import _ from 'lodash';
+
+import type { ValidationErrors } from '../flow/profileTypes';
+
+export default class ValidationAlert extends React.Component {
+  props: {
+    errors:   ValidationErrors,
+    children: React$Element[],
+  };
+
+  alertMessage: Function = (): React$Element|void => {
+    const { errors } = this.props;
+    if ( !_.isEmpty(errors) ) {
+      return (
+        <span className="message">
+          Please fix the errors above and try again
+        </span>
+      );
+    }
+  };
+
+  render () {
+    const { children } = this.props;
+    return (
+      <div className="validation-alert">
+        { this.alertMessage() }
+        <div className="actions">
+          { children }
+        </div>
+      </div>
```

Comment:
@gsidebo that pushes me towards leaving the `ValidationAlert` class as-is, because we can add it to those other non-dialog forms with one line. What do you think?